input job&data output wordlist

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import os
+import codecs
+import math
+import operator
+df = pd.read_csv('job_des.csv')
+def toword(txtlis):  #将切割成词表，返回list
+    wordlist = []
+    alltxt = ''
+    for i in txtlis:
+        alltxt = alltxt+str(i)
+    ridenter = alltxt.replace('\n',' ').replace('.',' ').replace('?',' ').replace('!',' ').replace(',',' ')
+    wordlist = ridenter.split()
+    return wordlist
+def getridofsw(lis, swlist):  #去除文章中的停用词
+    afterswlis = []
+    for i in lis:
+        if str(i) in swlist:
+            continue
+        else:
+            afterswlis.append(str(i))
+    return afterswlis
+def freqword(wordlis):  # 统计词频，并返回字典
+    freword = {}
+    for i in wordlis:
+        if str(i) in freword:
+            count = freword[str(i)]
+            freword[str(i)] = count+1
```

Comment:
suggestions: 
for i in wordlis:
    if str(i) in freword:
        freword[str(i)] += freword[str(i)] +1
    else:
        freword[str(i)] = 1
or:
from collections import Counter
freword = Counter()
for i in wordlis:
    freword[str(i)] += 1